ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/ReDeterminationMapper.java

```diff
@@ -16,19 +16,17 @@ public void to(CCDRespondent.CCDRespondentBuilder builder, Claim claim) {
         requireNonNull(builder, "builder must not be null");
         requireNonNull(claim, "claim must not be null");
 
-        claim.getReDetermination().ifPresent(reDetermination -> {
-            builder
-                .redeterminationExplanation(reDetermination.getExplanation())
-                .redeterminationMadeBy(CCDMadeBy.valueOf(reDetermination.getPartyType().name()));
-        });
+        claim.getReDetermination().ifPresent(reDetermination -> builder
+            .redeterminationExplanation(reDetermination.getExplanation())
+            .redeterminationMadeBy(CCDMadeBy.valueOf(reDetermination.getPartyType().name())));
 
         claim.getReDeterminationRequestedAt().ifPresent(builder::redeterminationRequestedDate);
     }
 
     public void from(Claim.ClaimBuilder builder, CCDRespondent respondent) {
         if (StringUtils.isBlank(respondent.getRedeterminationExplanation())
-            && respondent.getRedeterminationMadeBy() == null
-            && respondent.getRedeterminationRequestedDate() == null
+            || respondent.getRedeterminationMadeBy() == null
+            || respondent.getRedeterminationRequestedDate() == null
```

Comment:
Why was this change made? Is there a situation where only one of these could be blank/null?

Reply:
If there is, then we're now protected against NPEs and if there isn't then it's made no difference.

Reply:
Good spot @daveoh. I think I am happy this this explanation.